StepperEngine/src/Steps/CollectFilesInFolderStep.java

```diff
@@ -129,4 +131,17 @@ int addMatchingFilesToOutput(File[] files)
         return matchingFiles.getData().size();
     }
 
+    @Override
+    protected void updateStaticTimers() {
```

Comment:
can't this logic be implemented in the "Step" parent class? (same question for the 2 below methods as well)

Reply:
the ipplementation cannot be Step class because the fields updated in the method aren't Step's fields, they are his children's
fields, and thus he will not recognize those fields